lib/fog/vsphere/requests/compute/list_storage_pods.rb

```diff
@@ -0,0 +1,40 @@
+module Fog
+  module Compute
+    class Vsphere
+      class Real
+        def list_storage_pods(filters = { })
+          datacenter_name = filters[:datacenter]
+          raw_storage_pods(datacenter_name).map do |storage_pod|
+            storage_pod_attributes(storage_pod, datacenter_name)
+          end.compact
+        end
+
+        def raw_storage_pods(datacenter_name)
+          dc = find_raw_datacenter(datacenter_name)
+
+          @connection.serviceContent.viewManager.CreateContainerView({
+            :container  => dc,
+            :type       => ["StoragePod"],
+            :recursive  => true
+          }).view
+        end
```

Comment:
I believe this method `raw_storage_pods` should be private since its only purpose is to be used within `list_storage_pods`. Am i wrong?